packaging/windows/build_pypi_pkg/setup.py

```diff
@@ -0,0 +1,64 @@
+import sys
+import subprocess
+
+import setuptools.command.install
+from setuptools import find_packages, setup
+import wheel.bdist_wheel
+
+pkg_name="torch"
+pkg_ver="{{GENERATE_TORCH_PKG_VER}}"
+torch_win_url="https://download.pytorch.org/whl/torch_stable.html"
+
+
+python_min_version = (3, 6, 1)
+python_min_version_str = '.'.join((str(num) for num in python_min_version))
+python_max_version = (3, 9, 0)
+python_max_version_str = '.'.join((str(num) for num in python_max_version))
+
+class install_torch(setuptools.command.install.install):
+    def run(self):
+        setuptools.command.install.install.run(self)
+        raise Exception(
+            f"Can not install torch due to local wheel built failed."
+            f"Plesse visit {torch_win_url} for more details."
+        )
+
+class bdist_wheel(wheel.bdist_wheel.bdist_wheel):
+    def run(self):
+        subprocess.check_call(
+            [sys.executable,'-m', 'pip', 'download', f'{pkg_name}==={pkg_ver}', '-f', torch_win_url, '--no-deps', '-d', self.dist_dir])
+
+setup(
```

Comment:
need to specify `install_requires`, at least we need `wheel` to be there.
may also need to check min version required for wheel.

Reply:
Resolved

Reply:
we only need `wheel` here when installing our **fake** torch.
The **real** torch package already specified its **real** dependencies, so no need to include them here.